scripts/customise-flux-repositories.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
```

Comment:
quick description of what this is doing might be useful at the top of the script
(I think this is what it is doing, but it may be for the helm chart pulling)
```suggestion
# Adds service principal credentials to flux so that it can pull from our container registry
set -e
```